packages/gatsby-theme-newrelic/README.md

```diff
@@ -976,6 +981,38 @@ import { Button, CreateIssueButton } from '@newrelic/gatsby-theme-newrelic';
 />;
 ```
 
+### `DarkModeToggle`
+
+Used in combination with [`use-dark-mode`](https://www.npmjs.com/package/use-dark-mode), is an icon
+which sets a users' webpage to display either the light or dark theme
+
+```js
+import { DarkModeToggle } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Props**
+
+| Prop        | Type     | Required | Default | Description                                                       |
+| ----------- | -------- | -------- | ------- | ----------------------------------------------------------------- |
+| `className` | string   | no       |         | Optional className that can be added to the component             |
+| `onClick`   | function | no       |         | Addional functionality that can be added to the toggle if desired |
+| `size`      | string   | yes      |         | Size of the icon, must denote unit (e.g., `px`, `rem`, `etc`)     |
```

Comment:
It looks like `size` is not required on this component or any of the underlying components.
```suggestion
| `size`      | string   | no       |         | Size of the icon, must denote unit (e.g., `px`, `rem`, `etc`)     |
```